build.sbt

```diff
@@ -46,8 +46,10 @@ val scalacheckDep     = "org.scalacheck"                %% "scalacheck"
 val jolDep            = "org.openjdk.jol"                % "jol-core"                         % "0.9"
 val asmDep            = "org.scala-lang.modules"         % "scala-asm"                        % versionProps("scala-asm.version")
 val jlineDep          = "org.jline"                      % "jline"                            % versionProps("jline.version")
-val jansiDep          = "org.fusesource.jansi"           % "jansi"                            % versionProps("jansi.version")
-val jlineDeps         = Seq(jlineDep, jansiDep)
+val jlineTerminalDep  = "org.jline"                      % "jline-terminal"                   % versionProps("jline.version")
+val jlineJnaDep       = "org.jline"                      % "jline-terminal-jna"               % versionProps("jline.version")
```

Comment:
the `jline-terminal` and `jline-terminal-jna` deps should not be necessary, becaus we include the full `jline` jar that includes the other two, not the finer-graned jars (see https://github.com/jline/jline3#artifacts).

Reply:
That makes sense. I'll replace it by just adding JNA.

Reply:
ab802f7c1ac1cd746261748baba369e711e9e0fa